mtt.lua

```diff
@@ -1,14 +1,13 @@
-
 mtt.register("send mail", function(callback)
     -- create "player2"
     local auth_handler = minetest.get_auth_handler()
     auth_handler.set_password("player2", "")
 
     -- send a mail
-    mail.send("player1", "player2", "something", "blah")
+    mail.send({from = "player1", to = "player2", subject = "something", body = "blah"})
 
     -- check the receivers inbox
-    local list2 = mail.getMessages("player2")
+    local list2 = mail.getPlayerInboxMessages("player2")
```

Comment:
Normally, it should works, I think @BuckarooBanzay made it.

Reply:
@BuckarooBanzay for existing tests I think there's 2 good options:
* Either restore old tests and add new style / new API as a new tests so that both are tested.
* OR add deprecation warning for old API functions. (I would prefer this if new stuff seems better)
  * and possibly just delete stuff that probably isn't used even if public...